### ISSUE TITLE
Auto toggle between Autoconnect and AP?

### ISSUE THREAD

**Issue by Abhesheksh** [tzapu/WiFiManager#1375]
#### Hardware
WiFimanager Branch/Release:  Master

Esp8266/Esp32:  ESP8266

Hardware:			ESP-12e, esp01, esp25

ESP12E

Core Version:      	2.4.0, staging

### Description

In my case, the internet connection can go off for as long as 30-60 minutes, sometimes even more. Right now, after failing to connect, the ESP goes to AP mode and remains in AP mode forever. However, I would like it to change between AP mode and Autoconnect mode every 5-10 minutes. This is my relevant code right now below. What values should I use in setConfigPortalTimeout & setConnectTimeout to implement this?

  WiFi.mode(WIFI_STA);

  wifiManager.setConfigPortalTimeout(300); 
  wifiManager.setConnectTimeout(600); 
  
  wifiManager.setWiFiAPHidden(false);
  //wifiManager.setSTAStaticIPConfig(IPAddress(192, 168, 1, 99), IPAddress(192, 168, 1, 1), IPAddress(255, 255, 255, 0));
  wifiManager.autoConnect("ESP_WIFI_AP");

  if (wifiManager.autoConnect("ESP_WIFI_AP") == true)
  {
    Serial.println("Conneted to Internet");
    ticker.detach();
    digitalWrite(LED, LOW);

  }
  else
  {
    Serial.println("Connection Lost");
    send_email();
  }

### Settings in IDE

Module:  			NodeMcu, Wemos D1
NodeMCU

Additional libraries: 

#include <Ticker.h>
#include <ESP8266WiFi.h>
#include "ESP8266Ping.h"
#include <WiFiManager.h>
#include <EasyButton.h>
#include <NTPClient.h>
#include <WiFiUdp.h>

**Comment by Abhesheksh** [comment]
So basically, when wifi credentials are saved in the ESP but the saved network is switched off, it tries to autoconnect for a while but then goes to AP mode forever despite setting wifiManager.setConfigPortalTimeout(300) to 5 minutes. How do I make the ESP come back from AP mode and try connecting to the wifi network again?
